src/shared/components/header/index.js

```diff
@@ -7,33 +7,46 @@ import DesktopNavbar from 'shared/components/navbar/desktop'
 import MobileNavbar from 'shared/components/navbar/mobile'
 import styles from './index.module.css'
 
+const defaultScrollOptions = { offset: -66, align: 'top', duration: 800 }
+
 class Header extends Component {
   constructor () {
     super()
 
     this.state = {
-      isSticky: false
+      isSticky: false,
+      gettingStartedElement: undefined
     }
   }
 
+  componentDidMount () {
+    this.scrollToComponent = require('react-scroll-to-component')
+    this.setState({
+      gettingStartedElement: document.getElementById('gsContainer')
+    })
+  }
+
+  component
+
   render () {
     const localesBarHeight = this.localesBarElem && this.localesBarElem.clientHeight
     const { className } = this.props
     const { isSticky } = this.state
 
     return (
       <header className={ classNames(styles.header, className) }>
-        <Observer className={ styles.target } onChange={ this.handleObserverView }>
-          <span />
-        </Observer>
-        <DesktopNavbar isSticky={ isSticky } localesBarHeight={ localesBarHeight } />
-        <MobileNavbar />
+        <Observer className={ styles.target } onChange={ this.handleObserverView } />
```

Comment:
This `Observer` needs a child component, in order to work properly. I changed earlier [js.ipfs.io#64](https://github.com/ipfs/js.ipfs.io/pull/64) to this because this component was not working properly with Safari